### DISCUSSION TITLE
Update-Module: Module 'PnP.PowerShell' was not installed by using Install-Module, so it cannot be updated.

### DISCUSSION THREAD

**Discussion by nag433** [pnp/powershell#4135] [General]
unable to update pnp.powershell

**Comment by nag433** [comment]
Update-Module: Module 'PnP.PowerShell' was not installed by using Install-Module, so it cannot be updated.

**Comment by ToddKlindt** [comment]
Run this PowerShell to see where it's installed:
`Get-InstalledModule -Name pnp.powershell | select InstalledLocation`
You can manaully delete that directory in Explorer. Then you should be good to go.
